### PR TITLE
PADV-1474 feat: frontend - Create Gauge charts.

### PR DESCRIPTION
**Pull request by nandodev-net**
## Tickets

- https://agile-jira.pearson.com/browse/PADV-1474

## Description
This PR adds a React component named LabDetailsChartCard to display detailed information using gauge charts. The component should fetch data from the /details endpoint and use the MUI Charts library for rendering. The visual layout should be similar to the provided design, with two separate gauge charts representing tasks and scores.

## Changes Made

- [x] Updating react and react-dom to version 17
- [x] Add dependencies to mui charts: @mui/material @emotion/react @emotion/styled
- [x] Add @mui/x-charts library.
- [x] Add component LabDetailsChartCard to the LabDetails view.

## How To Test

Before any prior step, go to your MFE folder and execute `npm i` to install the new dependencies.

1. Please have your devstack environment running, run `npm start` and the MFE shall be running in port 2003 
2. go to the LMS and access to a skillable course.
3. Click on Training Lab tab to access the MFe.
4. Navigate from Strudent Roster to a Lab instance detail.
5. Verify the Gauge chart card.

Screenshot from 2024-08-08 15-59-41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,12 +37,16 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-platform": "4.5.1",
     "@edx/paragon": "20.45.0",
+    "@emotion/react": "11.13.0",
```
Comment (01001110J) [inline package.json:40]: Why is this for?

**Review by 01001110J — COMMENTED** [review]
What would happen if we upgrade the version of `react` and `react-dom` to the version `17`?

**Comment by nandodev-net** [comment]
> What would happen if we upgrade the version of `react` and `react-dom` to the version `17`?

react and react-dom updated to version 17

File: package.json [hunk 2]
```diff
@@ -37,19 +37,23 @@
...
+    "@mui/x-charts": "7.12.0",
     "core-js": "3.25.5",
     "jquery": "1.9.1",
     "jsoneditor": "^10.1.0",
     "popper.js": "1.16.1",
     "prop-types": "15.8.1",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```
Comment (01001110J) [inline package.json:56]:
```suggestion
    "react": "17.0.2",
    "react-dom": "17.0.2",
```

Please do not forget to update the package-lock.json as well


Are you 100% sure this change does not broke anything? Like Authentication or any different stuff of open edx

**Comment by 01001110J** [comment]
@nandodev-net could you please update the description about the upgrade of the React version.

File: package.json (hunk 2 above)
Comment (nandodev-net) [reply package.json:56]: @01001110J At least within the MFE everything seems to work well.

**Review by 01001110J — APPROVED** [review]

**Pull request merged**